test.disabled/ejabberd_tests/Makefile

```diff
@@ -20,6 +21,8 @@ endif
 
 COMMON_OPTS := -sname test -setcookie ejabberd -hidden \
                $(TLS_DIST_OPTS) \
+               -env REPO_DIR "$(shell readlink -f $(REPO_DIR))" \
```

Comment:
[We have a `readlink.sh` file stating that it's needed for MacOS](https://github.com/esl/MongooseIM/blob/master/readlink.sh) - how does this change work on Mac?

Reply:
Lovely, was thinking about this

```erlang
function readlink() {
  DIR=$(echo "${1%/*}")
  (cd "$DIR" && echo "$(pwd -P)")
}
```

Reply:
I don't know which is better: `readlink.sh` that we have in the repo or your proposed function, but I bet you can work it out :)

Reply:
This script looks broken. I will add another one into tools.

```erlang
./readlink.sh ../                                                          
/home/user/erlang/esl/2017/MongooseIM/.. 
```

Reply:
done